src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.DiagnosticState.cs

```diff
@@ -16,303 +16,305 @@
 
 namespace Microsoft.CodeAnalysis.Diagnostics.EngineV1
```

Comment:
I have this many times as well. Doing a merge before sending out PR seems to mess up the formatting.